dev-support/docker/Dockerfile_centos7

```diff
@@ -0,0 +1,96 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Could you please rename this file to `Dockerfile_centos_7`? Just so that we're consistent with the filename in trunk.

Reply:
@GauthamBanasandra I rename Dockerfile_centos7 to Dockerfile_centos_7 to keep consistent with the filename in trunk. and `mvn clean package -Dhttps.protocols=TLSv1.2 -DskipTests -Pnative,dist -Drequire.fuse -Drequire.openssl -Drequire.snappy -Drequire.valgrind -Drequire.zstd -Drequire.test.libhadoop -Pyarn-ui -Dtar -Dmaven.javadoc.skip=true > build.log 2>&1` logs here:  [build.log](https://github.com/apache/hadoop/files/7158247/build.log), and because some javadocs are illegal, and javadocs check failed, I add `-Dmaven.javadoc.skip=true` to build command. please review again, thanks.